tasks/collect-pyxis-params/collect-pyxis-params.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: collect-pyxis-params
   labels:
-    app.kubernetes.io/version: "0.1.1"
+    app.kubernetes.io/version: "0.2."
```

Comment:
forgot the 0

Reply:
Totally intentional
Should be fixed now